aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -220,19 +220,19 @@ Sample:
 The code above generates the following HTML:
 
 ```html
-  <form method="post" action="/Demo/RegisterInput">
-      Email:
-      <input type="email" data-val="true"
-             data-val-email="The Email Address field is not a valid email address."
-             data-val-required="The Email Address field is required."
-             id="Email" name="Email" value=""><br>
-      Password:
-      <input type="password" data-val="true"
-             data-val-required="The Password field is required."
-             id="Password" name="Password"><br>
-      <button type="submit">Register</button>
-      <input name="__RequestVerificationToken" type="hidden" value="<removed for brevity>">
-   </form>
+<form method="post" action="/Demo/RegisterInput">
```

Comment:
Fix the spacing (left new, right current):

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/be5dfe4f-4909-4ed5-be90-c143926e8fe1)